spec/requests/project_information_controller_spec.rb

```diff
@@ -25,7 +25,31 @@
     end
 
     it "returns a successful response" do
-      expect(subject).to have_http_status :success
+      expect(perform_request).to have_http_status :success
+    end
+
+    describe "links to assign users to project roles" do
+      let(:document) { Nokogiri.parse(body) }
```

Comment:
I am not sure what I think about forcing this request spec to be a feature spec by loading Nokogiri? Would we get the same confidence by expecting the unique paths to the change assignment to be in the body or not, something like:

```ruby
def has_change_links?
  expect(body).to include project_assign_team_lead_path(project) 
  expect(body).to include project_assign_regional_delivery_officer_path(project)
  expect(body).to include project_assign_caseworker_path(project)
end
```